src/test/kotlin/com/igorwojda/list/search/linearsearch/challenge.kt

```diff
@@ -4,7 +4,12 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun getIndex(list: List<String>, str: String): Int {
-    TODO("not implemented")
+    for (i in list.indices){
```

Comment:
You can also iterate with index and value: https://www.baeldung.com/kotlin/iterating-collections-by-index